### PR TITLE
Devel fixes

### PR DESCRIPTION
**Pull request by pconesa**

### CODE REVIEW

File: pyworkflow/tests/em/protocols/test_protocol_structure_mapping.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# ***************************************************************************
```
Comment (pconesa) [inline pyworkflow/tests/em/protocols/test_protocol_structure_mapping.py]: This test fails,
`Protocol failed: Command 'nma_record_info.py 20 pseudoatoms_1.pdb 15' returned non-zero exit status 127`

Comment (pconesa) [inline pyworkflow/tests/em/protocols/test_protocol_structure_mapping.py]: @mohsenkazemi can you double check this?.

Comment (mohsenkazemi) [inline pyworkflow/tests/em/protocols/test_protocol_structure_mapping.py]: @pconesa , I just double checked in asimov-dos and was working well!

>>>>> python  /home/mkazemi/scipion/pyworkflow/apps/pw_protocol_run.py "/home/mkazemi/ScipionUserData/projects/TestStructureMapping" "Runs/000082_XmippProtStructureMapping/logs/run.db" "82"
[ RUN   OK ] TestStructureMapping.testStructureMapping (88.485 secs)

[==========] run 1 tests (97.204 secs)
[  PASSED  ] 1 tests

Comment (pconesa) [inline pyworkflow/tests/em/protocols/test_protocol_structure_mapping.py]: I didn't have nma package instlalled, no idea it was a package.

**Pull request merged**